### PR TITLE
Add collector and analyzer to detect security tools

### PR DESCRIPTION
**Pull request by diamonwiggins**

### CODE REVIEW

File: host/default.yaml [hunk 1]
```diff
@@ -384,6 +384,10 @@ spec:
         collectorName: "ps-high-load"
         command: "sh"
         args: ["-c", "ps -eo s,user,cmd | grep ^[RD] | sort | uniq -c | sort -nbr | head -20"]
+    - run:
+        collectorName: "ps-detect-antivirus-and-security-tools"
+        command: "sh"
+        args: [-c, "ps -ef | grep -E 'clamav|sophos|esets_daemon|fsav|symantec|mfend|ds_agent|kav|bdagent|s1agent|falcon|illumio' | grep -v grep"]
```
Comment (banjoh) [inline host/default.yaml:390]: I'd have `ps -ef | grep -v grep | ...` to ensure you exclude grep process from the output

Comment (diamonwiggins) [reply host/default.yaml:390]: @banjoh I have it at the end of the command there to exclude it at the end. any reason it needs to be moved up? Not opposed to it. Just curious if there's something i'm missing

**Review by adamancini — APPROVED** [review]

File: host/default.yaml [hunk 2]
```diff
@@ -761,3 +765,15 @@ spec:
...
               message: "SELinux is disabled as expected"
+    - textAnalyze:
+        checkName: "Detect Threat Management and Network Security Tools"
+        fileName: host-collectors/run-host/ps-detect-antivirus-and-security-tools.txt
+        regex: '\b(clamav|sophos|esets_daemon|fsav|symantec|mfend|ds_agent|kav|bdagent|s1agent|falcon|illumio)\b'
+        ignoreIfNoFiles: true
+        outcomes:
+          - fail:
+              when: "true"
+              message: "Antivirus or Network Security tools detected. These tools can interfere with kubernetes operation."
```
[the remark this answers is not in the document]
Comment (chris-sanders) [reply host/default.yaml:776]: While I'm all for better visibility of this. EC runs actual preflights and I think kURL runs bash scripts. You might find you can't just add it to kURL.

[the remark this answers is not in the document]
Comment (diamonwiggins) [reply host/default.yaml:776]: Gotcha, yeah I think we'd have to add the ability to format the output a little better and then parse it out with some templating for this right? Feel free to create an issue for that.

**Pull request merged**